readme.md

```diff
@@ -268,13 +278,38 @@ d2 = asdict(o)
 assert d1 == d2
 ```
 
+Generators include the capability to construct their param-classes inline, if provided a set of compatible keyword arguments. For example, defining a generator using the `MyParams` parameters above:
+
+```python
+@h.generator
+def MyGen(params: MyParams) -> h.Module:
+    ... # Create a `Module` & return it
+```
+
+This typical invocation:
```

Comment:
Might be easier to explain by showing the explicit form used with a generator, ie.

```
p = MyParams(width=8, text="My Favorite Module")
My(p)
```
is the same as
```
MyGen(MyParams(width=8, text="My Favorite Module"))
```

Reply:
Having read #60, I can see that I've misunderstood what this part means.